.gitignore

```diff
@@ -1,3 +1,4 @@
 /vendor
 composer.phar
 composer.lock
+/.idea
```

Comment:
IDE specific folders should be excluded using a global gitignore file, see:

https://help.github.com/articles/ignoring-files/

Otherwise you'd have to take into consideration every single IDE any contributor ever made.